### PR TITLE
updates to campbeb3/add-bike

### PR DESCRIPTION
**Pull request by campbeb3**
made a few changes to the add-bike screen, but will need to rerun more after I gain back my Google auth login access.

### CODE REVIEW

File: bike_kollective/lib/add_bike_screen.dart [hunk 1]
```diff
@@ -1,80 +1,96 @@
...
-      return Bike(name: _name!, type: _type!, description: _description!, code: _code!);
+  void setImage(File image) {
+    _image = image; // Set selected image
+    notifyListeners(); // Notify listeners to update UI
+  }
+
+  BikeModel? get bike {
+    if (nameController.text.isNotEmpty && _type != null && descriptionController.text.isNotEmpty && lockCodeController.text.isNotEmpty) {
+      return BikeModel.newBike(
+        owner: BKDocumentReference.fake("ownerRef"),
```
Comment (ktucker720) [inline bike_kollective/lib/add_bike_screen.dart:47]: Get the active user from the provider, and pass in the active users docRef.

File: bike_kollective/lib/add_bike_screen.dart [hunk 2]
```diff
@@ -1,80 +1,96 @@
...
+        name: nameController.text,
+        type: _type!,
+        description: descriptionController.text, // Get description from its controller
+        code: lockCodeController.text, // Get lock code from its controller
+        imageLocalPath: '', // Use the imageLocalPath here if applicable
+        startingPoint: BKGeoPoint(0, 0),
```
Comment (ktucker720) [inline bike_kollective/lib/add_bike_screen.dart:53]: Use the user location provider data.

File: bike_kollective/lib/add_bike_screen.dart (hunk 1 above)
Comment (kenneth-tucker) [inline bike_kollective/lib/add_bike_screen.dart:47]: Comment about this placeholder being a temporary placeholder

File: bike_kollective/lib/add_bike_screen.dart (hunk 2 above)
Comment (kenneth-tucker) [inline bike_kollective/lib/add_bike_screen.dart:56]: Comment about this being a temporary placeholder.

Comment (kenneth-tucker) [reply bike_kollective/lib/add_bike_screen.dart:56]: You might consider a small class of your own for this, like BikeFormData. Then you can just have what you need. But this is ok given running out of time.

**Review by kenneth-tucker — APPROVED** [review]
Nice work! I left a couple style comments but these are optional and can be done later

**Review by anisar17 — APPROVED** [review]

**Pull request merged**